index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>WiFi4Kids</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h1 class="heading">Wifi4Kids</h1>
+        <a href="Features">Features</a>
+        <a href="Buy Now">Buy Now</a>
```

Comment:
You can use https://validator.w3.org/#validate_by_input+with_options to validate your HTML for any errors. 
An `href` can't have any spaces, found using the validator.

```suggestion
        <a href="Buy Now">Buy-Now</a>
```